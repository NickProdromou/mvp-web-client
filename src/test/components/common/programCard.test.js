import React from 'react';
import ProgramCard from '../../../components/common/programCard/programCard';
import renderer from 'react-test-renderer';

it('renders correctly', () => {

    const tree = renderer.create(
        <ProgramCard
            index={12}
            title="blah"
            keys={345}
            link={{to: '/a-location', text: 'SomeStuff'}}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
