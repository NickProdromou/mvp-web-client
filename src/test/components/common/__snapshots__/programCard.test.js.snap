// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-EHOje LlIEo sc-bxivhb jtJoDP"
>
  <article>
    <h3>
      blah
    </h3>
    <p>
      Keyboard shortcuts: 
      345
       
    </p>
    <a
      className="sc-bdVaJa bRdZkh"
      href="/a-location"
    >
      SomeStuff
    </a>
  </article>
</div>
`;
